qpmodel/Statis.cs

```diff
@@ -200,7 +200,7 @@ int compare(dynamic v1, dynamic v2)
                 if (which == -1) return otherfreq_;
                 return freqs_[which];
             }
-            
+
             double selectivity = calcTotalFreq(val, op);
```

Comment:
> double calcTotalFreq(Value val, string op) cannot handle this.

Have you tried lastest master code? It changed calcTotalFreq() to handle string/string comparison.